src/test/java/thread/ExecutorServiceFullCoreTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2017-2019 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package thread;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import java.util.concurrent.ThreadPoolExecutor;
+
+import org.junit.jupiter.api.Test;
+
+/*
+ * Tests for 30 Seconds of Java code library
+ *
+ */
+class ExecutorServiceFullCoreTest {
+  /**
+   * Tests for {@link ExecutorServiceFullCore#getExecutorFullCore()}.
+   */
+  @Test
+  public void testExecutorServiceFullCore() {
+    var core = Runtime.getRuntime().availableProcessors();
+    ThreadPoolExecutor executorService =
+            (ThreadPoolExecutor) ExecutorServiceFullCore.getExecutorFullCore();
+    assertEquals(core, executorService.getCorePoolSize());
```

Comment:
Check the parameters. I think its `expected, actual`, not the other way around.

Reply:
I know but number of processors return jvm. The test for any other machines will not pass when I use a concrete number. Expected is jvm value from availableProcessors method. Actual is the size of thread pool which is created by this value.

Reply:
Yes, you're right